app/queues/notifications/certificate-expiration-monitor-worker.server.ts

```diff
@@ -0,0 +1,53 @@
+import { Worker } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import type { DomainStatusData } from './record-cert-expiration-data.interface';
+import { addNotification } from './notifications.server';
+import { prisma } from '~/db.server';
```

Comment:
Please separate your imports into 3 sections:

```ts
// Third-party modules
import { Worker } from 'bullmq';
...

// Internal modules
import { redis } from '~/lib/redis.server';
...

// Types
import type ...
```

This makes the code much easier to read and understand where things are coming from, and how they are being used.